server/routes/induction/update/qualificationLevelUpdateController.ts

```diff
@@ -18,4 +14,24 @@ export default class QualificationLevelUpdateController extends QualificationLev
   getBackLinkAriaText(_req: Request): string {
     return 'Back to <TODO - check what CIAG UI does here>'
   }
+
+  submitQualificationLevelForm: RequestHandler = async (
+    req: Request,
+    res: Response,
+    next: NextFunction,
+  ): Promise<void> => {
+    const { prisonNumber } = req.params
+    const { prisonerSummary } = req.session
+
+    req.session.qualificationLevelForm = { ...req.body }
+    const { qualificationLevelForm } = req.session
+
+    const errors = validateQualificationLevelForm(qualificationLevelForm, prisonerSummary)
```

Comment:
This submission handler method doesn't do much, aside from validating the form and putting it in the session for the next screen (Qualification Details). We don't save the Induction during the flow, hence the removal of the `InductionService` above.